public/usage-examples/physics/scale_rotate_translate_matrix/scale_rotate_translate_matrix-simple.py

```diff
@@ -0,0 +1,56 @@
+from splashkit import *
+
+# Open a window for visualization
+open_window("Transformation Matrix Visualization", 400, 400)
+clear_screen(color_white())
+
+# Define the scaling factors
+matrix_scale = Point2D()
+matrix_scale.x = 1.5  # Scale width by 1.5
+matrix_scale.y = 1.2  # Scale height by 1.2
+
+# Define the translation (shift by a small amount)
+matrix_translation = Point2D()
+matrix_translation.x = 20  # Shift right by 20 units
+matrix_translation.y = -30 # Shift up by 30 units
+
+# Define the rotation angle
+rotation = 45
+
+# Create a matrix combining scaling, rotation, and translation
+transformation_matrix = scale_rotate_translate_matrix(matrix_scale, rotation, matrix_translation)
+
+# Define the original triangle points (centered and larger)
+original_triangle = Triangle()
+original_triangle.points[0] = Point2D()
+original_triangle.points[0].x = 200
```

Comment:
These next few lines are missing information comments that exist in the other files 
![image](https://github.com/user-attachments/assets/04df31b0-643d-4bb8-b7dc-93679f7d25e7)